launchable/commands/subset.py

```diff
@@ -294,14 +294,32 @@ def run(self):
 
             header = ["", "Candidates",
                       "Estimated duration (%)", "Estimated duration (min)"]
+            if "subset" not in summary.keys() or "rest" not in summary.keys():
+                click.echo(click.style(
+                    "Error: no subset or rest found.", 'yellow'), err=True)
```

Comment:
I think we don't have to show this message because already print an error message at https://github.com/launchableinc/cli/pull/377/files#diff-4c62e35fb4e3b2b6bff8593107cdddd1fd47eff90293edc9d838c9b2f3cff94aR279

What do you think?

Reply:
They are different. The exception is checking error for lanchable client, while this if clause is validating keys in exist in summary dictionary.

Reply:
Sorry, I want to confirm just in case.
Is there any case that even when Launchable returns status code 200, then the subset or the rest field doesn't exist?

Reply:
removed comment.
[eaa4f74](https://github.com/launchableinc/cli/pull/377/commits/eaa4f748ff8868e40e6d3400878391ba6a96708f)